lib/gui/app/components/drive-selector/controllers/drive-selector.js

```diff
@@ -75,17 +75,24 @@ module.exports = function (
       return $q.resolve(false)
     }
 
-    if (constraints.isDriveSizeRecommended(drive, selectionState.getImage())) {
-      return $q.resolve(true)
+    if (!constraints.isDriveSizeRecommended(drive, selectionState.getImage())) {
+      return WarningModalService.display({
+        confirmationLabel: 'Yes, continue',
```

Comment:
I wonder if this `'Yes, continue'` string should also be added to the messages class?